WolvenKit.RED4/Types/Classes/rendChunk.cs

```diff
@@ -45,7 +45,8 @@ public CArray<CName> MaterialId
 		}
 
 		[Ordinal(5)] 
-		[RED("vertexFactory")] 
+		[RED("vertexFactory")]
+		[DisplayAsEnum<EMaterialVertexFactory>]
```

Comment:
Should this maybe go into ClassesExt?

Reply:
It could do, but I was worried about obfuscation and confusion if someone in the future wanted to know why the fields were rendered as they were. I just needed some way to pass info from the RedClasses down the pipeline to the renderer without hardcoding it, an attribute with a strongly-typed param seemed the cleanest way to do that

Reply:
Its fine as is.
Doesn't make sense to put the attribute in ClassesExt

Reply:
Only thing which is questionable is, if its really needed to do add a generic attribute.